scripts/generate-selfsigned-ssl-certs.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+
+DOMAIN_NAME=$1
+if [ -z "${DOMAIN_NAME}" ]; then
+  echo "No domain-name was provided. Abnormal exit ..."
+  exit 1
+fi
+
+# Real path is not on every linux distribution.
+
+# SCRIPT_REAL_PATH=$(dirname $(realpath $0))
+SCRIPT_REAL_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
+
+pushd ${SCRIPT_REAL_PATH}
```

Comment:
It may be ok to just use "cd" here, but read comment further down on that issue and make a decision based on that.